### ISSUE TITLE
Typo in README.md: `argparser` instead of `argparse` + missing `ebooklib`

### ISSUE THREAD

**Issue by mattiasJohnson** [nikhil1raghav/kindle-send#1]
Great repo! A minor typo in the installation instructions: `argparser` is written instead of `argparse`.

**Comment by mattiasJohnson** [comment]
Corrected the typo #2

**Comment by mattiasJohnson** [comment]
When running the script I also got the following error which of course was mediated with installing `ebooklib`:

```python
 python kindle-send --link https://foreignpolicy.com/2022/08/26/europe-energy-crisis-natural-gas-economy-winter/
Traceback (most recent call last):
  File "/home/mattias/Downloads/programs-from-source/kindle-send/kindle-send", line 9, in <module>
    from utils.bundler import bundler
  File "/home/mattias/Downloads/programs-from-source/kindle-send/utils/bundler.py", line 4, in <module>
    from ebooklib import epub
ModuleNotFoundError: No module named 'ebooklib'
```

This was also fixed in my pull request (#2)
